plugins/rbac-backend/openapi.yaml

```diff
@@ -0,0 +1,636 @@
+                            type: string
+                          paramsSchema:
+                            type: object
+                          required:
+                            type: array
+                            items:
+                              type: string
+                          additionalProperties:
+                            type: boolean
+                          $schema:
+                            type: string
+        '403':
+          description: Refusal to authorize
+  /api/permission/roles/conditions:
+    get:
+      description: Lists all conditions
+      responses:
+        '200':
+          description: Request was successful
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: object
+                  items:
+                    type: object
+                    properties:
+                      id:
+                        type: integer
+                      result:
+                        type: string
+                      roleEntityRef:
+                        type: string
+                      pluginId:
+                        type: string
+                      resourceType:
+                        type: string
+                      permissionMapping:
+                        type: array
+                        items:
+                          type: string
+                      conditions:
+                        type: object
+                        properties:
+                          anyOf:
+                            type: array
+                            items:
+                              type: object
+                              properties:
+                                rule:
+                                  type: string
+                                resourceType:
+                                  type: string
+                                params:
+                                  type: object
+        '403':
+          description: Refusal to authorize
+    post:
+      description: Creates a new condition.
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                result:
+                  type: string
+                roleEntityRef:
+                  type: string
+                pluginId:
+                  type: string
+                resourceType:
+                  type: string
+                permissionMapping:
+                  type: array
+                  items:
+                    type: string
+                conditions:
+                  type: object
+                  properties:
+                    rule:
+                      type: string
+                    resourceType:
+                      type: string
+                    params:
+                      type: object
```

Comment:
```suggestion
                  $ref: '#/components/schemas/Condition'
```

Reply:
?

Reply:
Was having some issues committing my changes yesterday, fixed that now